quotecache/quotecache.go

```diff
@@ -75,7 +82,7 @@ func updateQuoteCache(userID, stock string) error {
 		return err
 	}
 
-	cache[stock] = quote
+	QuoteCache[stock] = map[string]Quote{userID: quote}
```

Comment:
You're right. I got lost in the `cache -> stocks -> users -> quotes` indirection!